.idea/codeStyles/Project.xml

```diff
@@ -18,15 +18,18 @@
       <option name="INSERT_INNER_CLASS_IMPORTS" value="true" />
       <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="100" />
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="100" />
-    <option name="IMPORT_LAYOUT_TABLE">
+      <option name="IMPORT_LAYOUT_TABLE">
         <value>
           <package name="" withSubpackages="true" static="true" />
           <emptyLine />
           <package name="" withSubpackages="true" static="false" />
         </value>
       </option>
     </JavaCodeStyleSettings>
-  <codeStyleSettings language="JAVA">
+    <JetCodeStyleSettings>
+      <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="2147483647" />
```

Comment:
intellij by default uses star imports, which ktlint does not like. This setting to intellij makes ktlint happy

Reply:
FWIW, wildcard imports can be disabled in Intellij: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

Reply:
right, that's what I did which caused this diff on this intellij file. I wanna say this sets the default so anyone else building this repo will get the same setting, so everyone wont need to go through and disable wildcard imports manually to get ktlint to pass, at least that would be the ideal case

Reply:
ooh that's great—thanks!